src/plugins/tracking/activity.js

```diff
@@ -117,13 +116,21 @@ export default function register(
 ) {
 	const { agent, isProd } = options;
 
-	const sessionIdCookieName: string = isProd ? 'SESSION_ID' : 'SESSION_ID_DEV';
-	const trackIdCookieName: string = isProd ? 'TRACK_ID' : 'TRACK_ID_DEV';
+	const trackIdCookieName: string = isProd
+		? 'MEETUP_TRACK'
```

Comment:
Note that we weren't actually using chapstick's `MEETUP_TRACK` cookie until now - we now consume it, and only create a new one if it doesn't exist in the initial request.